src/DistributionApi.php

```diff
@@ -208,9 +208,12 @@ private function createDownloadDirectoryIfNeeded(string $downloadPath): void
      */
     private function getResponse(string $endpoint): array
     {
-        /** @var array<array<string, string>> $response */
-        $response = json_decode($this->circruitBreaker->call($endpoint), true) ?: [];
+        $response = $this->circruitBreaker->call($endpoint, [], function () {
+            throw new PrestaShopException('Unable to retrieve informations from Distribution API : cannot automatically update native modules for the moment.');
+        });
+        /** @var array<array<string, string>> $json */
+        $json = json_decode($response, true) ?: [];
```

Comment:
```suggestion
        return json_decode($response, true) ?: [];
```

Reply:
Set result of json_decode is mandatory to make phpdoc and phpstan not failed, because json_decode return mixed